### PR TITLE
Add meme adder

### PR DESCRIPTION
**Pull request by VenatusSimpleX**
Will probably break bottu, pls abuse dis

Also, ignore my previous commits (if you can see them), I was an idiot (randomly pressing buttons).

### CODE REVIEW

File: Brain/ArchiBoT/Custom/vsx.siml [hunk 1]
```diff
@@ -19,5 +19,16 @@
       <Pattern>WHERE IS (VSX|VENA|VENATUS|VENATUSSIMPLEX)</Pattern>
       <Response>Kaleith: @vsx</Response>
     </Model>
+
+    <Model>
+      <Pattern>ADD MEME * (CALL|CALLED|NAME|NAMED|WITH LINK|LINK|LINKED) *</Pattern>
+      <Response>Meme added.</Response>
+      <Learn>
```
Comment (Thielak) [inline Brain/ArchiBoT/Custom/vsx.siml]: The `<Learn>` tag should be in the `<Response>`

File: Brain/ArchiBoT/Custom/vsx.siml [hunk 2]
```diff
@@ -19,5 +19,16 @@
...
+        <Model>
+          <Pattern><Process><Match /></Process></Pattern>
+          <Response><Process><Match At="2"/></Process></Response>
```
Comment (Thielak) [inline Brain/ArchiBoT/Custom/vsx.siml]: You probably want `<Match At="3"/>` here, as `<Match At="2"/>` would return whatever was used by the user in `(CALL|CALLED|NAME|NAMED|WITH LINK|LINK|LINKED)`

**Comment by VenatusSimpleX** [comment]
@Thielak Ye it's fixed, thanks for telling me and preventing bot breaku.

**Comment by Thielak** [comment]
I still think Pattern and Response should be switched !ai shrug

**Pull request merged**